tests/elastic_stabilization_time_scale_factor/screen-output

```diff
@@ -0,0 +1,80 @@
+-----------------------------------------------------------------------------
```

Comment:
Could you strip out the lines that start with `--`, `+` and `|` from this file?
Alternatively, if you have tested this prm using ctest, you can copy the file
 `aspect/build/tests/output-<test name>/screen-output.notime` into 
 `aspect/build/tests/<test name>/screen-output`.

Reply:
done :)